tests-web/www/index.html

```diff
@@ -36,13 +36,17 @@
     ></script>
 
     <script type="module" src="js/leaflet-osm.js"></script>
-    <script type="module" src="js/main.js"></script>
+    <script type="module">
+      import { StreetExplorer } from "./js/main.js";
+      window.app = await StreetExplorer.create(document.getElementById("map"));
+    </script>
   </head>
   <body>
     <menu>
       <h1>StreetExplorer</h1>
       <button type="button" id="import-view">Import current view</button>
       <ul id="test-list" />
+      <ul id="view-controls" />
```

Comment:
![Screenshot from 2022-08-01 12-32-11](https://user-images.githubusercontent.com/1664407/182139116-9fc35175-ac36-4f41-b7c8-8c5a786b29fb.png)
Why isn't this below the `test-list`? And even though I'm adding new child elements to it, it smushes the buttons together. CSS styling problem of some sort?